cpp/src/io/parquet/bloom_filter_reader.cu

```diff
@@ -599,55 +507,19 @@ std::vector<Type> aggregate_reader_metadata::get_parquet_types(
   return parquet_types;
 }
 
-std::pair<std::optional<std::vector<std::vector<size_type>>>, bool>
-aggregate_reader_metadata::apply_bloom_filters(
-  host_span<std::unique_ptr<datasource> const> sources,
+std::optional<std::vector<std::vector<size_type>>> aggregate_reader_metadata::apply_bloom_filters(
+  std::vector<rmm::device_buffer>& bloom_filter_data,
   host_span<std::vector<size_type> const> input_row_group_indices,
+  host_span<std::vector<ast::literal*> const> equality_literals,
   size_type total_row_groups,
   host_span<data_type const> output_dtypes,
-  host_span<int const> output_column_schemas,
+  host_span<int const> equality_col_schemas,
   std::reference_wrapper<ast::expression const> filter,
   rmm::cuda_stream_view stream) const
 {
   // Number of input table columns
   auto const num_input_columns = static_cast<cudf::size_type>(output_dtypes.size());
 
-  // Collect equality literals for each input table column
```

Comment:
Only do the filtering step here, bloom filter buffers are read in `predicate_pushdown.cpp` and passed here